src/components/illustration/illustration.tsx

```diff
@@ -0,0 +1,57 @@
+import IllustrationSet, { IllustrationType } from './illustration-set';
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+export enum IllustrationSize {
+  NORMAL = 'normal', // 72px
+  LARGE = 'large', // 144px
+  INHERIT = 'inherit', // 100% of the container
+}
+
+export interface IllustrationProps {
+  /**
+   * Pick an illustration by name.
+   */
+  show: IllustrationType;
+  /**
+   * Choose the size. Defaults to 72x72px, can be large (172x172px) or grow with the container.
```

Comment:
Large, according to the comments in `IllustrationSize` and the css, is `144x144px`. Would you please update this?

Reply:
Which is the correct size. I think the 72 bleed into my thoughts, thanks 😬